WTWSMS/common/laws/succession_laws.txt

```diff
@@ -661,7 +665,14 @@ succession_laws = {
 						is_republic = yes
 						AND = {
 							is_theocracy = yes
-							NOT = { holder_scope = { religion_group = muslim } }
+							NOT = { 
```

Comment:
Yes, but the code should be proper to avoid confusion. See the wiki recommendations, which I strive to respect.